src/python/pants/util/strutil_test.py

```diff
@@ -84,9 +84,9 @@ def test_strip_chroot_path() -> None:
         )
         == dedent(
             """\
-        Would reformat src/python/example.py
-        Would reformat test.py
-        Would reformat custom_tmpdir.py
+        Would reformat /<sandbox-1>/src/python/example.py
```

Comment:
Hmmm, looking at this I wonder if this may be suboptimal for a few reasons:

- editors that scan tooling output for paths are less likely to understand these as referring to something within the project being edited
- humans may be more confused e.g. someone who's just been told to use this "pants" tool by a team lead won't necessarily know where these non-existing paths are coming from
- any adhoc scripts that parse output may be broken

Some ideas, not mutually exclusive:

1. only add the disambiguation if there's more than one sandbox (doesn't fully solve the "can't trust the output" problem, though)
2. format it differently, e.g. `(sandbox-1) src/python/example.py` so that the sandbox disambiguation isn't part of the path
3. a reference to `pants` in the replacement, so it's obvious where it comes from (e.g. `pants-sandbox-1`)
4. Something else?

Thoughts?

Reply:
<sandbox-1> was meant to tackle 2, as it's clearly not-a-dir. I think (1) is a no-go from my POV, that's back to the original issue of "my printed env has the wrong paths?!?". (3) is fine, that's a minor edit to the wording. 

Other things... I don't have lots of good ideas. One could maybe imagine some blanking characters instead with full paths, or `/tmp/****/` -- pick a random blanking char -- but I imagine that's equally confusing. I did consider making full/partial blanking an option (i.e. pants.toml) but that would be some heavy refactoring at this point.

Reply:
Updated it to use `<pants-sandbox-##>` instead.